tasks/package.py

```diff
@@ -17,12 +16,26 @@ def get_package_path(glob_pattern):
     return package_paths[0]
 
 
+def _get_deb_uncompressed_size(ctx, package):
+    # the size returned by dpkg is a number of bytes divided by 1024
+    # so we multiply it back to get the same unit as RPM or stat
+    return int(ctx.run(f'dpkg-deb --info {package} | grep Installed-Size | cut -d : -f 2 | xargs').stdout) * 1024
```

Comment:
why do you need to pipe `xargs`?

Reply:
Because cut will also include the space before the size, and using xargs is the easiest way I know of to trim a string in shell.

It might work without, but this way I know any potential error isn't because of that space :)